src/main/java/io/stargate/sgv2/jsonapi/api/v1/GeneralResource.java

```diff
@@ -36,14 +37,15 @@
 @SecurityRequirement(name = OpenApiConstants.SecuritySchemes.TOKEN)
 @Tag(ref = "General")
 public class GeneralResource {
+  public static final String BASE_PATH = "/v1";
```

Comment:
nit: can we make the Keyspace and Collection resource use this constant to build it's path?